packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1663,6 +1663,7 @@ export class IntervalCollection<TInterval extends ISerializableInterval> extends
 		intervalType: IntervalType,
 		props?: PropertySet,
 	): TInterval {
+		assert(intervalType === IntervalType.SlideOnRemove, "interval type should be slide");
```

Comment:
A couple of SharedIntervalCollection tests fail on this assert - should I leave it in this PR or make a separate PR in next to make this change? I could also remove the type parameter all together if it goes into next.

Reply:
It's not related to the changes in this PR. I'd leave it out.